packages/loader/container-loader/src/container.ts

```diff
@@ -785,13 +804,6 @@ export class Container
 		// Prefix all events in this file with container-loader
 		this.mc = loggerToMonitoringContext(ChildLogger.create(this.subLogger, "Container"));
 
-		// Warning: this is only a shallow clone. Mutation of any individual loader option will mutate it for
```

Comment:
this is a behavior change that i don't think should be here

Reply:
No change, just moves up here: https://github.com/microsoft/FluidFramework/pull/15996/files#diff-c94f4bb2aaac11f9fc45cb8bfd7c09d091d54e38e437b727ac1244f1e8d5cb6cR749